app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -100,6 +102,10 @@
     public static final boolean DEBUG = !BuildConfig.BUILD_TYPE.equals("release");
     @NonNull
     public static final String TAG = "BasePlayer";
+    @NonNull
```

Comment:
Puth the keys for the shared preference into this file:
https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/res/values/settings_keys.xml